src/api-handler.ts

```diff
@@ -50,45 +66,46 @@ class APIResponse {
    *                   .contentType {string} Content-Type of response content
    *                   .content {string} Response content
    */
-  constructor(params) {
-    if (!params || !params.hasOwnProperty('status')) {
+  constructor(params: ResponseParams) {
+    if (!params || !params.status) {
       this.status = 500;
-      this.contentType = null;
-      this.content = null;
       return;
     }
 
     this.status = Number(params.status);
 
-    if (typeof params.contentType === 'undefined') {
-      this.contentType = null;
-    } else if (params.contentType !== null &&
-               typeof params.contentType !== 'string') {
+    if (params.contentType !== null &&
+      typeof params.contentType !== 'string') {
```

Comment:
According to the docs, it should be a string.
Is this really necessary?

Reply:
No, it's not. It was probably just a copy/paste error.